src/app.jsx

```diff
@@ -162,6 +166,9 @@ module.exports = React.createClass({
     });
     $.getJSON("serversetting.json", function( data ) {
       localStorage.setItem('server', data['server']);
+      self.state.logo = data['logo'] || null;
+      self.state.customStyle = data['customStyle'] || null;
+      self.state.links = data.links;
```

Comment:
Can you also add some defaults or just comments for the options in serversettings.json?

Reply:
Good point. I can't add comments to the default JSON file. But documenting it in README might work as well.

Reply:
> I can't add comments to the default JSON file.

Oh, yeah, I forgot how much JSON sucks :).